requirements.test.txt

```diff
@@ -1,5 +1,6 @@
 #Install the agent
-.
+-e .
+
```

Comment:
Not part of your change, but line 5 looks incorrect.  '@' instead if '#' before 'stable/liberty'?  I searched and didn't find use of '#' to specify branch or commit.